### PR TITLE
data: ElianCodes profile

### PR DESCRIPTION
**Pull request by ElianCodes**
Adds ElianCodes LinkFree profile

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8480"></a>

### CODE REVIEW

File: data/ElianCodes.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  "links": [
+    {
+      "name": "My Github profile",
+      "link": "https://github.com/eliancodes",
+      "icon": "FaGithub"
+    },
+
+    {
+      "name": "My website",
+      "link": "https://www.elian.codes",
```
Comment (sital002) [inline data/ElianCodes.json:43]:
```suggestion
      "url": "https://www.elian.codes",
```

File: data/ElianCodes.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    },
+    {
+      "url": "https://www.linkedin.com/in/elianvancutsem/",
+      "icon": "FaLinkedin"
+    }
+  ],
+  "links": [
+    {
+      "name": "My Github profile",
+      "link": "https://github.com/eliancodes",
```
Comment (sital002) [inline data/ElianCodes.json:37]:
```suggestion
      "url": "https://github.com/eliancodes",
```

**Review by sital002 — CHANGES_REQUESTED** [review]
link isn't a valid property It should be url.

**Review by sital002 — APPROVED** [review]
Looks Good!

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for Contributing! 🎉 

<p align="center"><a href="https://linkfree.io/docs/how-to-guides/editing"></a></p>

**Pull request merged**